mule-extensions-api/src/main/java/module-info.java

```diff
@@ -150,9 +150,7 @@
       org.mule.runtime.extensions.api.test;
 
   exports org.mule.runtime.extension.internal.util to
-      org.mule.runtime.extensions.support,
```

Comment:
IIRC, I added these because the IDE was marking these as missing and generating compilation issues. Has the underlyaing problem been solved?

Reply:
That one wasn't removed yet, however I'm working on removing its usages so it's no longer needed.